packages/blockchain-wallet-v4-frontend/src/data/components/interest/sagas.ts

```diff
@@ -113,7 +113,7 @@ export default ({
     }
   }
 
-  const fetchInterestLimits = function * ({
+  const fetchInterestLimits = function*({
```

Comment:
why are all these formatting changed?  Seems like a lint rule difference, update your branch with latest `development` and address